corehq/apps/data_cleaning/templates/data_cleaning/columns/task_form_ids.html

```diff
@@ -0,0 +1,5 @@
+{% if record.has_form_ids %}
+  <a href="{{ record.form_ids_url }}" target="_blank">
+    <i class="fa fa-cloud-download"></i>
```

Comment:
technically this is still the "old style" font awesome name for the icon. could you update it to use the new name and `fa-solid`? We have the "transition" setting enabled for the library, but at some point they will probably remove the old references in a later version.
see https://fontawesome.com/icons/cloud-arrow-down?f=classic&s=solid

Reply:
sure - I'll do that one in a followup, I can do it across the repo